packages/starter/example/components/ContextProvider.tsx

```diff
@@ -59,15 +59,20 @@ const WalletContextProvider: FC<{ children: ReactNode }> = ({ children }) => {
 
     const wallets = useMemo(
         () => [
-            /**
-             * Select the wallets you wish to support, by instantiating wallet adapters here.
-             *
-             * Common adapters can be found in the npm package `@solana/wallet-adapter-wallets`.
-             * That package supports tree shaking and lazy loading -- only the wallets you import
-             * will be compiled into your application, and only the dependencies of wallets that
-             * your users connect to will be loaded.
-             */
-            new FakeWalletAdapter(),
+            new WalletConnectWalletAdapter({
```

Comment:
i'll remove this and put it into a README.md

Reply:
Cool! Yeah, I'd say just move the example stuff to the `packages/wallets/walletconnect` README probably. https://github.com/solana-labs/wallet-adapter/pull/498 removed all preference for wallets, but it would be fine (good, even!) for individual adapter readmes to show how to use them, especially since this one requires config.